.github/workflows/stdlib-workflow-trigger.yml

```diff
@@ -0,0 +1,22 @@
+name: Stdlib Dependency Update
+
+on:
+  push:
```

Comment:
Shall we add the `branches` filter as well? (Eg.: https://github.com/ballerina-platform/module-ballerina-graphql/blob/70d904b3eb142ac0de32ca4e4267e6e68282cf5d/.github/workflows/stdlib-workflow-trigger.yml#L5)

Reply:
Added in 3677c7b